R/create_hexsticker.R

```diff
@@ -0,0 +1,38 @@
+#' @title Make hexagonal logo for package
+#'
+#' @description This function makes a hexsticker in INBO style for the provided
+#' package name.  It requires the package hexSticker to be installed.
```

Comment:
This is me nitpicking. Can you start a new sentence on a new line? That keeps the diffs smaller as it will list only the changes sentences.